lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -497,7 +500,7 @@ def self.available_options
 
           FastlaneCore::ConfigItem.new(key: :group,
                                   env_name: "APPCENTER_DISTRIBUTE_GROUP",
-                               description: "Distribute group name",
+                               description: "Comma separated list of distribute group names",
```

Comment:
Can you please change the description to `Comma separated list of Distribution Group names`. I know it was wrong before already, but why not doing it here and now :)